stratum/hal/bin/barefoot/start-stratum.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+# Copyright 2020-present Open Networking Foundation
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+KDRV_PATH=${KDRV_PATH:-/usr/lib/modules/bf_kdrv.ko}
+PLATFORM=${PLATFORM:-x86-64-accton-wedge100bf-32x-r0}
+
+if [ -f "/etc/onl/platform" ]; then
+    PLATFORM="$(cat /etc/onl/platform)"
+fi
+
+# Setup port map
+rm /usr/share/port_map.json || true
+ln -s /etc/stratum/stratum_configs/$PLATFORM/port_map.json /usr/share/port_map.json
+
+# Load Kernel module
+rmmod bf_kdrv || true
+insmod $KDRV_PATH intr_mode="msi"
```

Comment:
This will fail if we are using docker images like `stratumproject/stratum-bf:9.1.0` since we don't include kernel module in it.

Reply:
I see.